src/main/java/bio/terra/service/job/JobMapKeys.java

```diff
@@ -10,6 +10,7 @@
     STATUS_CODE("status_code"),
     AUTH_USER_INFO("auth_user_info"),
     SUBJECT_ID("subjectId"),
+    CLOUD_PLATFORM("cloudPlatform"),
```

Comment:
This should really not be an enum (compare with `FileMapKeys`, `LoadMapKeys`, etc). more tech debt I suppose.